src/core_modules/capture-core/components/ScopeSelector/QuickSelector/OrgUnitSelector.component.js

```diff
@@ -1,185 +1,77 @@
-    listItem: {
-        backgroundColor: '#ffffff',
-        borderRadius: 5,
-        marginBottom: 5,
-        padding: 5,
-        border: '1px solid lightGrey',
-    },
-    selectedText: {
-        marginTop: 5,
-        marginBottom: 5,
-        padding: 5,
-        borderLeft: '2px solid #71a4f8',
-    },
-    selectedPaper: {
-        backgroundColor: theme.palette.grey.lighter,
-        padding: 8,
-    },
-    selectedButton: {
-        width: 20,
-        height: 20,
-        padding: 0,
-    },
-    selectedButtonIcon: {
-        width: 20,
-        height: 20,
-    },
-    selectedItemContainer: {
-        display: 'flex',
-        alignItems: 'center',
-        minHeight: 28,
-        marginTop: 8,
-        marginBottom: 4,
-        paddingLeft: 5,
-        borderLeft: `2px solid ${theme.palette.primary.light}`,
-    },
-    selectedItemClear: {
-        flexGrow: 1,
-        display: 'flex',
-        justifyContent: 'flex-end',
+const styles = () => ({
+    selectBarMenu: {
+        minWidth: '60vw',
+        maxHeight: '90vh',
+        overflow: 'scroll',
+        minHeight: spacers.dp96,
     },
 });
-
 type Props = {
-    handleClickOrgUnit: (orgUnitId: ?string, orgUnitObject: ?Object) => void,
+    handleClickOrgUnit?: (orgUnitId: ?string, orgUnitObject: ?Object) => void,
     onReset: () => void,
-    selectedOrgUnitId: string,
-    showWarning: boolean,
-    selectedOrgUnit: Object,
+    selectedOrgUnitId?: string,
+    selectedOrgUnit?: {
+        name: string
+    },
     previousOrgUnitId?: string,
     classes: Object,
 };
 
 type State = {
-    open: boolean,
+   open: boolean,
 };
 
 class OrgUnitSelectorPlain extends Component<Props, State> {
-    handleShowWarning: () => void;
     handleClose: () => void;
     handleClick: (orgUnit: Object) => void;
-    handleReset: () => void;
-    constructor(props) {
+    constructor(props: Props) {
         super(props);
 
         this.state = {
             open: false,
         };
 
-        this.handleShowWarning = this.handleShowWarning.bind(this);
-        this.handleClose = this.handleClose.bind(this);
         this.handleClick = this.handleClick.bind(this);
-        this.handleReset = this.handleReset.bind(this);
-    }
-
-    handleShowWarning() {
-        if (this.props.showWarning) {
-            this.setState({ open: true });
-        } else {
-            this.handleReset();
-        }
-    }
-
-    handleClose() {
-        this.setState({ open: false });
     }
 
-    handleClick(selectedOu) {
+    handleClick(selectedOu: {id: string, displayName: string, code?: string}) {
         const orgUnitObject = { id: selectedOu.id, name: selectedOu.displayName, code: selectedOu.code };
-        this.props.handleClickOrgUnit(selectedOu.id, orgUnitObject);
-    }
-
-    handleReset() {
-        this.props.handleClickOrgUnit(undefined, null);
+        const { handleClickOrgUnit } = this.props;
+        handleClickOrgUnit && handleClickOrgUnit(selectedOu.id, orgUnitObject);
```

Comment:
I've noticed that when you are in an enrollment and trying to select an orgUnit, the window does not automatically close. Is this intended or a bug?